credentials/src/test/kotlin/web5/sdk/credentials/VerifiableCredentialTest.kt

```diff
@@ -94,40 +101,75 @@ class VerifiableCredentialTest {
   }
 
   @Test
-  fun `signing works`() {
-    val keyManager = InMemoryKeyManager()
-    val issuerDid = DidKey.create(keyManager)
-    val holderDid = DidKey.create(keyManager)
-
-    val vc = VerifiableCredential.create(
-      type = "StreetCred",
-      issuer = issuerDid.uri,
-      subject = holderDid.uri,
-      data = StreetCredibility(localRespect = "high", legit = true)
+  fun createJwk() {
+    val privateJwk = JWK.parse(
+      """{
+          "kty": "EC",
+          "d": "OeBCt3M8roz9F9Ny192TtZXf_-qPzAkwSM6ep7QdSO4",
+          "use": "sig",
+          "crv": "secp256k1",
+          "kid": "NB_qlUIr06-AikVVMFreq0lc-omQtzc6lwhhcvgO6r4",
+          "x": "DdtN8W6x_34pB_nkxR0e1tmDkNnsJeusBAEPzKWgf_Y",
+          "y": "u3W135inodLqtcEb9jNGS3JsM_uFKmkJSb8Trc9luWI",
+          "alg": "ES256K"
+        }"""
```

Comment:
I'm not seeing this being used anywhere. Still needed?

Reply:
It wasn't, thanks for catching! Removed